src/Domain/Services/ExtendRestApi.php

```diff
@@ -52,7 +53,7 @@ public function get_formatter( $name ) {
 	 *
 	 * @var array
 	 */
-	private $endpoints = [ CartItemSchema::IDENTIFIER, CartSchema::IDENTIFIER ];
+	private $endpoints = [ CartItemSchema::IDENTIFIER, CartSchema::IDENTIFIER, CartExtensionsSchema::IDENTIFIER ];
```

Comment:
`CartItemSchema::IDENTIFIER` and `CartSchema::IDENTIFIER` are endpoints you can add data to, `CartExtensionsSchema::IDENTIFIER` is the endpoint that will get the data, it doesn't make sense for it to be here.

Reply:
Agreed, removed.